ingredients/emaildelivery/main.go

```diff
@@ -25,73 +24,27 @@ func MakeIngredient(config TypeInputWithService) Ingredient {
 }
 
 func SendSMTPEmail(config SMTPServiceConfig, content SMTPGetContentResult) error {
+	m := gomail.NewMessage()
```

Comment:
How is the new way of sending being tested?

Reply:
Manually Gmail SMTP settings. both 587 & 465.

Reply:
Also, content override with IsHtml true and false.